tests/zstreamtest.c

```diff
@@ -548,6 +548,34 @@ static int basicUnitTests(U32 seed, double compressibility)
     {   size_t const ret = ZSTD_decompressStream(zd, &outBuff, &inBuff);
         if (ret != 0) goto _output_error;
     }
+
+    {
```

Comment:
Please add a title for this test like the previous one. You should probably edit the title of the previous one as well, so it's clear what scenarios are being tested in both cases.